apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -366,15 +383,20 @@
           "default": 1
         },
         "advanceScorePerc": {
-          "$ref": "#/definitions/advanceScorePerc"
+          "description": "Minimum score percentage to unlock access to subsequent questions",
+          "$ref": "#/definitions/AdvanceScorePercJsonSchema"
+        },
+        "singleVariant": {
+          "description": "Whether the question is not randomized and only generates a single variant (defaults to \"false\").",
+          "type": "boolean"
```

Comment:
👀 this looks like a brand-new property, was adding this intentional?

Reply:
It was not, changed in [0ffc275](https://github.com/PrairieLearn/PrairieLearn/pull/11024/commits/0ffc275517f833b3c018439522c3485474571243).

Reply:
It is declared like this here: https://github.com/PrairieLearn/PrairieLearn/blob/772ef771a208bbb10f20e9ad34db941c1657221e/exampleCourse/courseInstances/SectionA/assessments/gallery/elements/infoAssessment.json#L28

Reply:
So I'm not sure what is the correct behavior: is this example incorrect, or is the schema incorrect?

Reply:
That example is almost definitely incorrect, but you should check the implementation as the source of truth to be sure. From a quick glance, this is not supported, as `single_variant` is only a column on the `questions` table, not `assessment_questions` or anything else.